test/selections-from-parent-dir.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+cd "${CURR_DIR}/selections-from-parent-dir/pkg2" || die "Could not cd."
+
+"$DUB" build --nodeps
+
+if [ -f dub.selections.json ]; then
+    die "Shouldn't create a dub.selections.json in pkg2 subdir."
+fi
```

Comment:
Any chance you could convert this to the new test framework ? E.g. see `source/dub/test/other.d`.

Reply:
I did; pretty cool. :+1: - `dub test -c library-nonet` fails for some rather bizarre reason (the CI problem), `dub test` alone works fine.

Reply:
Solved; was a problem of the mocked file system (no entries for `.`  and `..`, so needs extra normalization; might be required in other places too).